config/environment.rb

```diff
@@ -1,7 +1,19 @@
 # frozen_string_literal: true
 
+LOG_LEVELS = ['debug', 'info', 'warn', 'error', 'fatal', 'unknown']
+LOG_LEVEL_ERROR = "Environment variable 'CONJUR_LOG_LEVEL' must be a valid Rails log level: #{LOG_LEVELS.inspect}"
+
+def assert_valid_conjur_log_level
+  log_level_env = ENV['CONJUR_LOG_LEVEL']
+  if log_level_env
+    raise LOG_LEVEL_ERROR unless LOG_LEVELS.include? log_level_env
```

Comment:
How would the user correct an invalid log level?  Would they have to restart Rails?  Will this exception kill the Rails process (I forget, as it's been a while)?  We have to make sure that this is properly documented in @sigalsax's new doc, so that it gets in front of customers.

Reply:
Yes, they will have to restart rails. this error will occur on rails initialization so it's fine that it kills the process. the user tried to run rails and failed with a descriptive message